interface/forms/sdoh/table.sql

```diff
@@ -0,0 +1,138 @@
+CREATE TABLE IF NOT EXISTS `form_sdoh` (
+id bigint(20) NOT NULL auto_increment,
+date datetime default NULL,
+pid bigint(20) default NULL,
+user varchar(255) default NULL,
+groupname varchar(255) default NULL,
+authorized tinyint(4) default NULL,
+activity tinyint(4) default NULL,
+education varchar(30),
+disability varchar(30),
+housing varchar(30),
+housingotherinput varchar(30),
```

Comment:
for these *otherinput entries, will be better to use `text` than `varchar(30)` for 2 reasons:
1. Then information doesn't get truncated from the form (if a users inputs something that is over 30 characters)
2. Decreases the size that the table takes up to avoid going above the max limit (if a user decides to add to the form and this table)